getting-started/documentation.html

```diff
@@ -33,7 +32,7 @@
         <ul class="list">
             <div class="list-tile font-primary font-x-large fw-600">Getting Started</div>
             <li class="list-item">
-                <a class="btn btn-link-secondary btn-focus-primary w-100 justify-content-start font-large p-y-0_5">Installation</a>
+                <a class="btn btn-link-secondary btn-focus-primary w-100 justify-content-start font-large p-y-0_5" href="#installation">Installation</a>
             </li>
         </ul>
         <p> </p>
```

Comment:
This is an empty tag that can be removed.

Reply:
ok sure